cypress/e2e/inventory/holdings/locations-temporary-validate-matching-settings.cy.js

```diff
@@ -0,0 +1,137 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import generateItemBarcode from '../../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
+import { ITEM_STATUS_NAMES } from '../../../support/constants';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import HoldingsRecordView from '../../../support/fragments/inventory/holdingsRecordView';
+import HoldingsRecordEdit from '../../../support/fragments/inventory/holdingsRecordEdit';
+import { Locations } from '../../../support/fragments/settings/tenant';
+import settingsMenu from '../../../support/fragments/settingsMenu';
+
+describe('Temporary Location', () => {
```

Comment:
please use two describes like
describe('inventory', () => {
describe('Item', () => {
and put all tests to 'e2e\inventory\item' folder